src/ace/managers/system.c

```diff
@@ -342,6 +344,25 @@ static void msgPortDelete(struct MsgPort *mp) {
 	FreeMem(mp, sizeof(*mp));
 }
 
+static void ioRequestInitialize(struct IORequest *pIoReq, struct MsgPort *pMsgPort, UWORD uwLen) {
+	pIoReq->io_Message.mn_Node.ln_Type = NT_MESSAGE;
+	pIoReq->io_Message.mn_ReplyPort = pMsgPort;
+	pIoReq->io_Message.mn_Length = uwLen;
+}
+
+static struct IORequest *ioRequestCreate(struct MsgPort *pMsgPort, UWORD uwLen) {
+	struct IORequest *pIoReq = (struct IORequest *) AllocMem(uwLen, MEMF_PUBLIC | MEMF_CLEAR);
+	if (!pIoReq) {
+		return NULL;
+	}
+	ioRequestInitialize(pIoReq, pMsgPort, uwLen);
+	return pIoReq;
+}
+
+static void ioRequestDelete(struct IORequest *pIoReq) {
```

Comment:
I think this will be more consistent with ACE naming scheme
```suggestion
static void ioRequestDestroy(struct IORequest *pIoReq) {
```